tests/DependencyInjection/HandlerMapping/TestMappingData.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace League\Tactician\Bundle\Tests\DependencyInjection\HandlerMapping;
+
+final class TestMappingData
```

Comment:
Hmm, doesn't look like this class is used anywhere anymore?

Reply:
you are right ;)

Reply:
As they say, "Even a blind pig finds an acorn once in a while!" 🐖